src/modules/expression/unop/neg.rs

```diff
@@ -1,8 +1,14 @@
-use heraclitus_compiler::prelude::*;
-use crate::{utils::{metadata::ParserMetadata, TranslateMetadata}, modules::types::{Type, Typed}, translate::{module::TranslateModule, compute::{translate_computation, ArithOp}}};
-use super::{super::expr::Expr, UnOp};
 use crate::docs::module::DocumentationModule;
 use crate::error_type_match;
+use crate::modules::expression::expr::Expr;
```

Comment:
I think we should avoid nested `use` statements like the one I'm replacing here, as they're hard to read and maintain. RustRover has handy tools to "flatten use statements" and "optimize imports".